spec/controllers/user_sessions_controller_spec.rb

```diff
@@ -2,6 +2,10 @@
 
 require 'spec_helper'
 
+# We actually need to use multiple befores as part of our fake
+# login
+# rubocop:disable RSpec/ScatteredSetup
+
```

Comment:
No need for the scattered setup. To fix this, you can:
- move `before { activate_authlogic }` to the top-level block, as it makes sense to have that active for any specs for this controller:
```ruby
describe UserSessionsController do
  before { activate_authlogic }
```
- remove that line from elsewhere in the specs
- make the request into a `subject`:
```ruby
subject(:destroy) { delete :destroy }
```
That will ensure the controller method is implicitly called by the examples, without needing a separate `before`.

Reply:
I tried this during development, and ran into problems. I don't have the test runs to hand, so I'll re-insert the change and see if they recur.

Reply:
Changed.